docs/4.2/enterprise/ssh_sso.md

```diff
@@ -34,6 +34,15 @@ succeeds, Teleport will retrieve an SSH certificate and will store it in
 `~/.tsh/keys/proxy.example.com` directory and also will add it to an 
 [SSH agent](https://en.wikipedia.org/wiki/Ssh-agent) if there's one running.
 
+### SSO Setup Guides
+
+- [Azure Active Directory (AD)](sso/ssh_azuread.md)
+- [Active Directory (ADFS)](sso/ssh_adfs.md) 
+- [G Suite](sso/ssh_gsuite.md)
+- [OneLogin](sso/ssh_one_login.md)
+- [OIDC](sso/oidc.md)
+- [Okta](sso/ssh_okta.md)
+
```

Comment:
We should move this section up as close to the top of the page as possible. It's in danger of disappearing off the bottom even at 2560x1440:

![2020-05-12-104139__1746x1294__maim](https://user-images.githubusercontent.com/897821/81699317-8c8a2100-943d-11ea-8588-54a4952bf66a.png)

Reply:
I've moved this up and also remove the very large introduction text as it was redundant.